scripts/tiqit/__init__.py

```diff
@@ -998,7 +998,7 @@ def getBasePath():
     return basepath
 
 def getBaseHost():
-    return 'http://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
+    return 'https://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
```

Comment:
Should 'siteurl' (and the other example URLs) in the config files (config & doc/examples/tiqit.conf) be updated too?